test/TestProjects/MgmtMockTest/tests/Generated/CodeModel.yaml

```diff
@@ -7641,6 +7641,7 @@ operationGroups:
                 resourceGroupName: sample-resource-group
                 subscriptionId: 00000000-0000-0000-0000-000000000000
                 vaultName: sample-vault
+              x-ms-original-file: file:///C:/ZZ/projects/codegen/autorest.csharp/test/TestProjects/MgmtMockTest/src/examples/createVault.json
```

Comment:
Something in the testmodeler might need change - we should not output an absolute path here (reason please see above comments)